constants.py

```diff
@@ -62,8 +96,30 @@
     "cam10": False,
 }
 
-annotated_section_start = vid_timecodes['vid2_seg8']
-annotated_section_end = vid3_start + timedelta(minutes=12)
-vid3_offset_from_vid2 = vid3_start - annotated_section_start
-annotated_section_len = (annotated_section_end - annotated_section_start)
+annotated_section_start = cam4_vid_timecodes["vid2_seg8"]
+annotated_section_end = CAM4_VID3_START_TIMECODE + timedelta(minutes=12)
+vid3_offset_from_vid2 = CAM4_VID3_START_TIMECODE - annotated_section_start
+annotated_section_len = annotated_section_end - annotated_section_start
+
+# Paths to the raw dataset in the staff-bulk storage, mounted in the current PC.
+RAW_VIDEOS_FOLDER_IN_STAFF_BULK = Path(
+    "/mnt/staff-bulk/ewi/insy/SPCDataSets/conflab-mm/raw/video/overhead/"
+)
+VIDEO_SEGMENTS_FOLDER_IN_STAFF_BULK = Path(
+    "/mnt/staff-bulk/ewi/insy/SPCDataSets/conflab-mm/processed/annotation/videoSegments/"
+)
+SYNCED_AUDIO_FOLDER_IN_STAFF_BULK = Path(
+    "/mnt/staff-bulk/ewi/insy/SPCDataSets/conflab-mm/release/release-final/data-raw/audio/synced/"
+)
+
+def check_if_staff_bulk_is_mounted():
+    if not RAW_VIDEOS_FOLDER_IN_STAFF_BULK.exists():
+        raise FileNotFoundError(
+            "Mount the bulk storage first in /mnt/staff-bulk\nUse: sshfs -o ro NETID@sftp.tudelft.nl:/staff-bulk /mnt/staff-bulk/"
+        )
```

Comment:
Would it make sense to put create a new var like so:
```python
STAFF_BULK = Path("/mnt/staff-bulk")
```
to avoid duplication?

Reply:
Sure, in this context, it makes sense to use the entire path up to conflab  `/mnt/staff-bulk/ewi/insy/SPCDataSets/conflab-mm/`

Reply:
nevermind. I think your suggestion is better, if what we want to emphasize is where to mount